piwik-sdk/src/main/java/org/piwik/sdk/DownloadTracker.java

```diff
@@ -50,20 +70,25 @@ public DownloadTracker(Tracker tracker, TrackMe baseTrackMe) {
         mContext = piwik.getContext();
         mPackageName = mContext.getPackageName();
         mPackMan = mContext.getPackageManager();
+        try {
+            PackageInfo pkgInfo = mPackMan.getPackageInfo(mContext.getPackageName(), 0);
+            mVersion = getVersion();
+            if (mVersion == null){
+                throw new PackageManager.NameNotFoundException("Version could not be found");
```

Comment:
This `try/catch` block and those one above at `L56` looks very similar. It's better to refactor them and follow DRY principle